plugin/completion.py

```diff
@@ -162,13 +154,12 @@ def on_modified(self):
                 if self.response_incomplete:
                     # debug('incomplete, triggering new completions')
                     self.view.run_command("hide_auto_complete")
-                    sublime.set_timeout(self.run_auto_complete, 0)
```

Comment:
This might seems strange, but what this does is it makes sure that `self.run_auto_complete` is ran from Sublime's main thread. I am unsure wether that is necessary though.

Remember that these "handle response" methods always run on a Python thread.

Reply:
what happens if it doesn't run in sublime's main thread ?

Reply:
I don't fully understand the consequences of that. So just to be safe, my suggestion is to revert this line.

Reply:
Ok I've reverted this line